AAUnlimited/Texts/mod/subtitles.lua

```diff
@@ -1,10 +1,24 @@
 --@INFO Subtitles loader
 
 local _M = {}
---local opts = {
---	{ "maxlines", 4, "Maximum number of lines to show %i[1,10]"},
---	{ "duration", 3, "Subtitles duration %i[1,10]:" },
---}
+local opts = {
+	{ "fontfam", 0, "Font family: %l|Arial|Tahoma|" },
```

Comment:
This should be a text box

Reply:
And what happens if the user enters the wrong value?
Easier to add a variety of standard fonts in this list. What do you think about this?

Reply:
Arial and Tahoma is a very limited selection for everyone. Not only for aesthetic reasons but because those fonts don't support all languages like Chinese, Japanese or Korean. There must be a textbox with some default value that lets you select any other font. If the users enters the wrong value, that's the users fault, not ours. Arial Unicode MS could work as a default.

Please try %n instead and tell me if the font picker works for you.